client_apis/python/src/tests/integration/alerts_test.py

```diff
@@ -0,0 +1,212 @@
+#
+# CARBON BLACK API TESTS - alerts
+# Copyright, Bit9, Inc 2015
+#
+
+""" These tests require CarbonBlack Enterprise Server to be installed.
+    You can run this script by passing the server URL and user API token
+    as parameters.
+"""
+
+import unittest
+import sys
+import os
+import json
+import requests
+import time
+import paramiko, base64
+
+
+if __name__ == '__main__':
+    sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../")))
+
+from cbapi.cbapi import CbApi
+from datetime import datetime
+
+cb = None
+
+class ShhHelper:
```

Comment:
I added a helpers subpackage to `tests/integration`, so this helper would belong there as well, especially, since it could/would be used by other tests